geometry_msgs/Point last_waypoint_in_neighborhood
int16 out_direction
